lib/wget.js

```diff
@@ -82,18 +83,28 @@ function download(src, output, options, _parentEvent, redirects) {
                 download(newLocation, output, options, downloader, redirects);
             }
 
-            if (res.statusCode === 200) {
+            if (res.statusCode === 200 || res.statusCode === 206) {
```

Comment:
Do we have the http status code library installed here in upload? If it is, maybe we should use hat to make this more readable/understandable?

Reply:
No http-status-code library in node-wget, unfortunately.
Should we add it? Or should I just probably add a comment explaining 206.

Reply:
I think we should I guess to standardize it on our node projects as welll

Reply:
Done